rxbinding/src/main/java/com/jakewharton/rxbinding/view/RxView.java

```diff
@@ -19,6 +21,18 @@
  */
 public final class RxView {
   /**
+   * Create an observable of attach and detach events on {@code view}.
+   * <p>
+   * <em>Warning:</em> The created observable keeps a strong reference to {@code view}. Unsubscribe
+   * to free this reference.
+   */
+  @CheckResult
+  public static Observable<ViewAttachEvent> attachEvents(View view) {
```

Comment:
I mean, you can claim `filter` for _every_ binding we have but then any action you take has to allocate a wrapper object and for particularly chatty events that cost will start to add up as your compose more and more bindings together. My struggle between these here is real.